app/presenters/completions_presenter.rb

```diff
@@ -105,8 +106,12 @@ def pii
     end
   end
 
-  def multiple_emails?
-    current_user.confirmed_email_addresses.many?
+  def email_change_link
```

Comment:
I think the idea is that we'll _always_ send the user to `sign_up_select_email_path`, even if they have a single email address. So we might not need this logic after all.

Reply:
I thought the idea was to keep the logic as is, just the language change. 
AC listed in https://cm-jira.usa.gov/browse/LG-14998
> The "Add new email" link is replaced with "Change" and the workflow remains the same for users with a single email on their Login.gov account as previously.

Reply:
Hm, that's not what I'd expected when proposing this change, though I think you're right to what's written in the ticket. Let's check with the team.

Reply:
We talked with team offline and discussed that it should take the user to the sign_up_select_email_path always. Updated code to reflect that AC.